sentry-core/src/main/java/io/sentry/core/Scope.java

```diff
@@ -278,6 +281,24 @@ public void removeExtra(@NotNull String key) {
     this.extra.remove(key);
   }
 
+  /**
+   * Returns the Scope's contexts
+   *
+   * @return the contexts
+   */
+  public @NotNull Contexts getContexts() {
```

Comment:
I know I originally added this but I'm not sure it's a good idea to expose the setter/getter.
It could be needed later to have a `clear` method on the scope in which case we'd set a new instance of Contexts but that could all be internal.

Reply:
just in case it needs to be public, we can add `@ApiStatus.Internal` at least.
but I guess in this case package-private would be enough

Reply:
now reading again, if we don't make this public, we won't be able to offer statically typed getters and setters like
`Device getDevice()` and `void setDevice(Device device)` on the scope, as only the Contetxts class has these methods.

or we could at least to expose:
setContexts(String key, Object value), also a getter, we'd need to solve that either way.

Reply:
My comment should have gone on the `setContexts`, the getter is fine as you said to get the typed context stuff